.github/actions/linux/build-libmongocrypt/action.yml

```
@@ -0,0 +1,29 @@
+name: "Build libmongocrypt"
+description: "Installs libmongocrypt"
+inputs:
+  version:
+    description: "Libmongocrypt version to install (major.minor)"
+    required: true
+runs:
+  using: composite
+
+  steps:
+    - name: Add repository key
+      shell: bash
+      run: sudo sh -c 'curl -s --location https://pgp.mongodb.com/libmongocrypt.asc | gpg --dearmor >/etc/apt/trusted.gpg.d/libmongocrypt.gpg'
+
+    - name: Add repository
+      shell: bash
+      working-directory: /tmp
+      # Note: no packages for Ubuntu 24.04 noble exist, so we use those for 22.04
```

```suggestion
      # Note: no packages for Ubuntu 24.04 noble exist, so we use those for 22.04. Update after MONGOCRYPT-813 is released.
```

Aside: MONGOCRYPT-813 is planned for the next 1.14.1 patch release.

---

Thank you, I'll update this alongside 1.14.1 then :)